pyproject.toml

```diff
@@ -114,4 +114,6 @@ filterwarnings = [
     # TODO: if these ignores are needed, enable them, otherwise remove them
```

Comment:
`'ignore:"@coroutine" decorator is deprecated since Python 3\.8, use "async def" instead:DeprecationWarning',` can also be removed